### ISSUE TITLE
Text elements must have sufficient color contrast against the background

### ISSUE THREAD

**Issue by GGD4V** [MeD-DMC/adapt-contrib-esdc-college-theme#26]
Issue: Blue headings (headings of aria-level="2") on their pale gray background fail minimum contrast ratio. How to fix: I would suggest tuning down the colour to #1C4E8A
